nova/files/liberty/nova-compute.conf.Debian

```diff
@@ -182,6 +182,10 @@ catalog_info=volumev2:cinderv2:internalURL
 
 {%- if compute.get('ceph', {}).ephemeral is defined %}
 [libvirt]
+disk_cachemodes="network=writeback,block=none"
+cpu_mode=host-model
+virt_type=kvm
+live_migration_flag=VIR_MIGRATE_UNDEFINE_SOURCE,VIR_MIGRATE_PEER2PEER,VIR_MIGRATE_LIVE,VIR_MIGRATE_PERSIST_DEST
```

Comment:
live_migration_flag
We have this already in DEFAULT section bug with +TUNNELED.

cpu_mode=host-model
Any idea what differences and what specific related to ceph an option has? We already have libvirt_cpu_mode=host-passtrough vs. this?